src/box/lua/config/descriptions.lua

```diff
@@ -0,0 +1,2446 @@
+    not maintained. A node with wal.mode set to none can't be a replication
+    master. write: fibers wait for their data to be written to the
+    write-ahead log (no fsync(2)). fsync: fibers wait for their data,
+    fsync(2) follows each write(2).
+
+]])
+
+M['wal.queue_max_size'] = format_text([[
+    The size of the queue in bytes used by a replica to submit new transactions
+    to a write-ahead log (WAL). This option helps limit the rate at which a
+    replica submits transactions to the WAL. Limiting the queue size might be
+    useful when a replica is trying to sync with a master and reads new
+    transactions faster than writing them to the WAL.
+]])
+
+M['wal.retention_period'] = format_text([[
+    The delay in seconds used to prevent the Tarantool garbage collector from
+    removing a write-ahead log file after it has been closed. If a node is
+    restarted, `wal.retention_period` counts down from the last modification
+    time of the write-ahead log file.
+
+    The garbage collector doesn't track write-ahead logs that are to be
+    relayed to anonymous replicas, such as:
+
+    - Anonymous replicas added as a part of a cluster configuration
+    (see `replication.anon`).
+    -CDC (Change Data Capture) that retrieves data using anonymous replication.
+
+    In case of a replica or CDC downtime, the required write-ahead logs
+    can be removed. As a result, such a replica needs to be rebootstrapped.
+    You can use wal.retention_period to prevent such issues.
+
+    Note that `wal.cleanup_delay` option also sets the delay used to prevent
+    the Tarantool garbage collector from removing write-ahead logs. The
+    difference is that the garbage collector doesn't take into account
+    `wal.cleanup_delay` if all the nodes in the replica set are up and running,
+    which may lead to the removal of the required write-ahead logs.
+]])
+
+-- }}} wal configuration
+
+-- }}} Instance descriptions
+
+-- {{{ Cluster descriptions
+
+local L = {}
+
+L[''] = 'Cluster configuration'
+
+-- {{{ conditional configuration
+
+L['conditional'] = format_text([[
+    The `conditional` section defines the configuration parts that apply to
+    instances that meet certain conditions.
+]])
+
+L['conditional.*'] = format_text([[
+    `conditional.if`: Defines a section where configuration options apply only
+    if specified conditions are met.
+
+    Conditions can include `tarantool_version` and support comparison operators
+    (`>`, `<`, `>=`, `<=`, `==`, `!=`), as well as logical operators
+    (`||`, `&&`) and parentheses for grouping.
+]])
+
+L['conditional.*.*'] = format_text([[
+    `conditional.if.*`: Defines a specific condition and the configuration
+    options applied when the condition is met.
+
+    Each condition compares the `tarantool_version` using operators
+    (`<`, `>`, `<=`, `>=`, `==`, `!=`) and supports defining labels or other
+    options that are applied only if the condition holds true.
+]])
```

Comment:
Can we just say something like the following?

> `if` or a cluster configuration field.

Reply:
Of course we can) Replaced with:
```lua
C['conditional.*.*'] = format_text([[
    `if` or a cluster configuration field.
]])
```